util/opentelemetry-util-tracer/src/opentelemetry/util/tracer/__init__.py

```diff
@@ -0,0 +1,48 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opentelemetry import trace
+from opentelemetry.trace.span import Span
+from typing import Optional 
+from opentelemetry.util import types
+
+class ServerSpanTracer:
+    "Decorator class for tracer for conditionally creating server spans"
```

Comment:
Also wasn't sure about this until now, but it seems like this should be surrounded by triple quotes, even though it's still on one line 👍 [One-liner Docstrings](https://www.python.org/dev/peps/pep-0257/#id16). See for example [this](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/9c834f088150e208d4fbd189477ed3bca5d106a3/opentelemetry-instrumentation/src/opentelemetry/instrumentation/propagators.py#L100)?
```suggestion
    """Decorator class for tracer for conditionally creating server spans."""
```